cypress/e2e/fee-fine/lost-item-require-actual-cost.cy.js

```diff
@@ -0,0 +1,211 @@
+import uuid from 'uuid';
+
+import { getTestEntityValue } from '../../support/utils/stringTools';
+import { DevTeams, Permissions, TestTypes } from '../../support/dictionary';
+import TopMenu from '../../support/fragments/topMenu';
+import Users from '../../support/fragments/users/users';
+import InventoryInstances from '../../support/fragments/inventory/inventoryInstances';
+import { ServicePoints } from '../../support/fragments/settings/tenant';
+import { Locations } from '../../support/fragments/settings/tenant/location-setup';
```

Comment:
Can be imported from index file:

import { Locations, ServicePoints } from '../../support/fragments/settings/tenant';